changelog.d/13768.misc

```diff
@@ -0,0 +1 @@
+Port push rules to using Rust.
```

Comment:
I think we've done some speed-ups in the past a feature.

Reply:
I don't actually think this will make things go faster by itself tbh

Reply:
Guess we could combine it if we land more improvements.